### DISCUSSION TITLE
Requests are not logged when not running with flask run

### DISCUSSION THREAD

**Discussion by denisvitez** [pallets/flask#4713] [Q&A]
Hey,

I recently upgraded my application from Flask `v1.x` to `v2.1.3`. And since I'm using some 3rd party library that no longer supports running applications with `flask run` I am now running the application as `python app.py`. I am wondering what I have to do to get the old logging behavior since logging of all requests is really beneficial for debugging process.  I tried searching for this information but was so far unsuccessful. If someone knows what to do or can point me to somewhere where this has already been answered I will be very grateful :)

**Comment by davidism** [comment]
The development server logs requests to stderr automatically. If you're using something else, such as a production server, you'll need to look at that server to configure its logging.

**Reply by denisvitez** [reply]
Hey,

hmm, I think it should be running as a development server I'm just running it directly not with `flask run`. This is the output that I get when the application is started:

```
* Restarting with stat
* Debugger is active!
* Debugger PIN: 109-071-781
```
All exceptions and such things still get logged to the terminal normally, only the requests are missing so I'm thinking maybe `flask run` uses some special configuration for that?

**Reply by davidism** [reply]
I can't reproduce that issue with the information provided. Request lines are logged in both 2.1 and 2.2 for me. You'll need to update your question with a minimal reproducible example.
